src/main/java/com/swervedrivespecialties/exampleswerve/subsystems/DrivetrainSubsystem.java

```diff
@@ -3,65 +3,68 @@
 import com.revrobotics.CANSparkMax;
 import com.revrobotics.CANSparkMaxLowLevel;
 import com.swervedrivespecialties.exampleswerve.RobotMap;
-import com.swervedrivespecialties.exampleswerve.commands.DriveCommand;
+import com.swervedrivespecialties.exampleswerve.constants.ChassisConstants;
```

Comment:
You might want to think about just doing

```suggestion
import com.swervedrivespecialties.exampleswerve.constants.ChassisConstants.*;
```

doing that will simply import all of the ChassisConstants things without you having to reference them as ChassisConstants.whatever. you can just use the variable WHATEVER for every such variable in the class and it'll know what you mean. There are reasons not to do it, but it could also simplify things

Reply:
rip should have looked at this before changing